### ISSUE TITLE
Nova 4 Support

### ISSUE THREAD

**Issue by beshoo** [marshmallow-packages/nova-tool-data-importer#29]
Dos it support Nova 4?

**Comment by stefvanesch** [comment]
@beshoo it doesnt at the moment. Did you find another solution for Nova 4?

**Comment by beshoo** [comment]
@stefvanesch
Unfortunately...

**Comment by stefvanesch** [comment]
I will look into how much work it is to upgrade this to Nova 4. You still need this correct?

**Comment by beshoo** [comment]
To be honest with you, this is from long time, and I did my own way, but yes the open-source community needs it. And I may need it, so its good to follow-up with latest nova as long as it will not consume all your time.

**Comment by stefvanesch** [comment]
Yes i noticed this issue just now. Thats my bad. Good to here that you have found a solution. I will add this to my to do to make this available on Nova 4.
